collection-scripts/gather_monitoring

```diff
@@ -78,7 +104,7 @@ monitoring_gather(){
   prom_get status/config status/config || true
   prom_get status/flags status/flags || true
   prom_get status/runtimeinfo status/runtimeinfo || true
-  prom_get status/tsdb status/tsdb || true
+  prometheus_gather_by_replica || true
```

Comment:
@dgrisonnet   could be, but I did in this way to avoid N*path OC calls to discovery the Prometheus' pod here:
``` shell
prometheus_gather_by_replica(){
  echo "INFO: Collecting status for each replica..."
  local pods="$(oc get pods -n openshift-monitoring -l prometheus=k8s -o jsonpath='{.items[*].metadata.name}')"
```

Changed the way, ptal?